components/tooltip/stories/template.js

```diff
@@ -12,6 +12,7 @@ export const Template = ({
 	rootClass = "spectrum-Tooltip",
 	label,
 	placement,
+	variant = "neutral",
```

Comment:
informative & negative got added back to Figma 11/18/24.

Reply:
The gradient test case is failing VRT currently, so we'll need to implement the `background-color` > `background` CSS property changes. 

Original fix: https://github.com/adobe/spectrum-css/pull/2929